website_sale_tax_toggle/demo/assets.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<data>
+    <template id="assets_frontend_demo" inherit_id="website.assets_frontend">
```

Comment:
```suggestion
    <template id="assets_tests_demo" inherit_id="website.assets_tests">
```
On v13 exists the the view assets_test. This assets are loaded just when testing, so you can place the file on templates and inherit from this view on data. Not need to place the assets on demo anymore